examples/simple_module/src/main.py

```diff
@@ -36,7 +36,7 @@ def validate_config(cls, config: ComponentConfig) -> Sequence[str]:
                 raise Exception("Multiplier cannot be 0.")
         else:
             cls.multiplier = 1.0
-        return [""]
+        return []
```

Comment:
Good catch; was this causing implicit deps bugs?

Reply:
yep